bundles/org.openhab.core/src/main/java/org/eclipse/smarthome/core/library/items/CallItem.java

```diff
@@ -28,7 +28,7 @@
  * This item identifies a telephone call by its origin and destination.
  *
  * @author Thomas.Eichstaedt-Engelen / Initially CallItem
- * @author Gaël L'hopital - port to Eclipse SmartHome
+ * @author Gaël L'hopital - port to openHAB
```

Comment:
Hm, I assume this cannot be changed as it will change the "history".
Gaël L'hopital has been ported it to Eclipse SmartHome, regardless if we migrate that code from Eclipse SmartHome to openHAB Core.